src/System.Private.ServiceModel/tests/Common/Infrastructure/src/testproperties.props

```diff
@@ -1,74 +1,67 @@
-        <DefaultValue>8083</DefaultValue>
-      </TestProperty>
-      <TestProperty Include="BridgeSecureWebSocketPort">
-        <Value>$(BridgeSecureWebSocketPort)</Value>
-        <DefaultValue>8084</DefaultValue>
-      </TestProperty>
-      <TestProperty Include="BridgeRemoteEnabled">
-        <Value>$(BridgeRemoteEnabled)</Value>
-        <DefaultValue>false</DefaultValue>
-      </TestProperty>
-      <TestProperty Include="BridgeCertificatePassword">
-        <Value>$(BridgeCertificatePassword)</Value>
-        <DefaultValue>test</DefaultValue>
-      </TestProperty>
-      <TestProperty Include="BridgeCertificateValidityPeriod">
-        <Value>$(BridgeCertificateValidityPeriod)</Value>
-        <DefaultValue>7.00:00:00</DefaultValue>
-      </TestProperty>
-      <TestProperty Include="UseFiddlerUrl">
-        <Value>$(UseFiddlerUrl)</Value>
-        <DefaultValue>false</DefaultValue>
-      </TestProperty>
-      <TestProperty Include="BridgeMaxIdleTimeSpan">
-        <Value>$(BridgeMaxIdleTimeSpan)</Value>
-        <DefaultValue>24:00:00</DefaultValue>
-      </TestProperty>
-      <TestProperty Include="MaxTestTimeSpan">
-        <Value>$(MaxTestTimeSpan)</Value>
-        <DefaultValue>00:01:00</DefaultValue>
-      </TestProperty>
-    </ItemGroup>
+    <PropertyGroup Condition="'$(BridgeResourceFolder)' == ''">
+      <BridgeResourceFolder>$(OutputPath)</BridgeResourceFolder>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeHost)' == ''">
+      <BridgeHost>localhost</BridgeHost>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgePort)' == ''">
+      <BridgePort>44283</BridgePort>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeHttpPort)' == ''">
+      <BridgeHttpPort>8081</BridgeHttpPort>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeHttpsPort)' == ''">
+      <BridgeHttpsPort>44285</BridgeHttpsPort>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeTcpPort)' == ''">
+      <BridgeTcpPort>809</BridgeTcpPort>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeWebSocketPort)' == ''">
+      <BridgeWebSocketPort>8083</BridgeWebSocketPort>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeSecureWebSocketPort)' == ''">
+      <BridgeSecureWebSocketPort>8084</BridgeSecureWebSocketPort>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeRemoteEnabled)' == ''">
+      <BridgeRemoteEnabled>false</BridgeRemoteEnabled>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeCertificatePassword)' == ''">
+      <BridgeCertificatePassword>test</BridgeCertificatePassword>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BridgeCertificateValidityPeriod)' == ''">
+      <BridgeCertificateValidityPeriod>7.00:00:00</BridgeCertificateValidityPeriod>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(UseFiddlerUrl)' == ''">
+      <UseFiddlerUrl>false</UseFiddlerUrl>
+    </PropertyGroup>
```

Comment:
Correct, but I'd like to avoid mixing up PR's with orthogonal purposes, especially because they propagate to changes in the BridgeProperties.  I can make a separate PR to remove this without much trouble.